### ISSUE TITLE
New Program input errors improvments

### ISSUE THREAD

**Issue by shanemc-goog** [civiform/civiform#1699]
Input errors on the New Program page are hard to follow

Currently they are appended into a long string and shown as a toast that disappears quickly.

We could make that nicer or switch to error messaging on the relevant input fields as we do in the Applicant flow for filling out programs.
![new program errors](https://user-images.githubusercontent.com/90928876/141596056-9269ec12-1cb7-47c8-a1cb-f65d4e53f468.png)

**Comment by kristinyim** [comment]
Related to #2004
